README.md

```diff
@@ -1,15 +1,64 @@
+6. Bitcoin Mailing list: [Data Source](https://lists.linuxfoundation.org/pipermail/) -- [Cron Job Source](https://github.com/bitcoinsearch/scraper/blob/master/.github/workflows/mailing-list-bitcoin.yml)
+    - **schedule**: Every Day at 00:00 (Mid Night)
+    - The purpose of this cron job is to automate the process of fetching data from the Bitcoin mailing list and indexing it into ElasticSearch. By running this job daily at midnight, users can ensure that their ElasticSearch index remains up-to-date with the latest discussions and announcements from the Bitcoin community.
+
+7. Lightning Mailing list: [Data Source](https://lists.linuxfoundation.org/pipermail/lightning-dev/) -- [Cron Job Source](https://github.com/bitcoinsearch/scraper/blob/master/.github/workflows/mailing-list-lighting.yml)
+    -  **schedule**: Every Day at 00:00 (Mid Night)
+    - This cron job is designed to fetch posts and discussions from the Lightning Development mailing list hosted at https://lists.linuxfoundation.org/pipermail/lightning-dev/ and store them in an ElasticSearch index. The Lightning Development mailing list serves as a crucial platform for discussions and development updates related to the Lightning Network, a caching layer for Bitcoin, making this cron job essential for staying informed about the latest developments in Lightning Network technology.
+    - The purpose of this cron job is to automate the process of fetching posts and discussions from the Lightning Development mailing list and indexing them into ElasticSearch. By running this job regularly, users can ensure that their ElasticSearch index remains updated with the latest discussions, proposals, and development updates from the Lightning Network community.
+8. StackExchange : [Data Source](https://lists.linuxfoundation.org/pipermail/lightning-dev/) -- [Cron Job Source](https://github.com/bitcoinsearch/scraper/blob/master/.github/workflows/stackexchange.yml)
+    -  **schedule**: Every Sunday at 00:00 (Mid Night)
+    - This Python-based cron job is designed to fetch discussions related to Bitcoin from the StackExchange platform and store them in an ElasticSearch index. StackExchange is a popular platform for asking questions, sharing knowledge, and engaging in discussions on various topics, including Bitcoin, making this cron job essential for aggregating and indexing valuable insights from the Bitcoin community.
+    - The purpose of this cron job is to automate the process of fetching Bitcoin-related discussions from StackExchange and indexing them into ElasticSearch. By running this job every Sunday at midnight, users can ensure that their ElasticSearch index remains updated with the latest discussions and insights from the StackExchange Bitcoin community.
+
```

Comment:
I appreciate the effort to update the README.

I think that Cron Jobs section is too repetitive regarding what each cron job is doing without providing any substantial information about how it is doing it. I also think that the information highlighting what each source is all about (like describing what StackExchange is) are not needed here.

I also had an update for this README on draft which I will share below as a more concise and imo more appropriate alternative to what you wrote here:
```suggestion
This project is designed to automate the process of gathering information from a variety of key Bitcoin-related sources.
It leverages GitHub Actions to schedule nightly cron jobs, ensuring that the most up-to-date content is captured from each source according to a defined frequency.
The scraped data are then stored in an Elasticsearch index.

Below is a detailed breakdown of the sources scraped and the schedule for each:

Daily at 00:00 AM UTC
- [Lightning Mailing List](https://lists.linuxfoundation.org/pipermail/lightning-dev/) ([cron](.github/workflows/mailing-list-lightning.yml), [source](mailing-list))
- [New Bitcoin Mailing List](https://gnusha.org/pi/bitcoindev/) ([cron](.github/workflows/mailing-list-bitcoin-new.yml), [source](mailing-list/main.py))
- [Bitcoin Mailing List](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/) ([cron](.github/workflows/mailing-list-bitcoin.yml), [source](mailing-list))
- [Delving Bitcoin](https://delvingbitcoin.org/) ([cron](.github/workflows/delving-bitcoin.yml), [source](delvingbitcoin_2_elasticsearch))

Weekly
- [bitcoin.stackexchange](https://bitcoin.stackexchange.com/) ([cron](.github/workflows/stackexchange.yml), [source](bitcoin.stackexchange.com))
- Bitcoin Talk Forum ([cron](.github/workflows/bitcointalk.yml), [source](bitcointalk))
    - only the [Development & Technical Discussion Board](https://bitcointalk.org/index.php?board=6.0)
    - only for specific authors
- [Bitcoin Transcript](https://btctranscripts.com/) ([cron](.github/workflows/bitcointranscripts.yml), [source](bitcointranscripts))
- [Bitcoin Optech](https://bitcoinops.org/) ([cron](.github/workflows/bitcoinops.yml), [source](bitcoinops))

Additionally, for on-demand scraping tasks, we utilize a Scrapybot, details of which can be found in the [Scrapybot section](#scrapybot) below.

```

Reply:
Hi @kouloumos Thank you for your insights, Let's move forward with your version of ReadMe file.